bindata/assets/kube-apiserver/pod.yaml

```diff
@@ -127,6 +127,8 @@ spec:
         valueFrom:
           fieldRef:
             fieldPath: status.hostIP
+      - name: GOGC
+        value: {{ printf "%q" .GOGC }}
```

Comment:
naive question, why can not this be templated directly and we have to use a printf?

Reply:
It's related reasoning to https://github.com/openshift/cluster-kube-apiserver-operator/pull/1359#discussion_r896925534. This mostly avoids encoding too much information about GOGC into the template, since there's precedent for acceptable non-integer GOGC values. `"{{ .GOGC }}"` would produce invalid YAML if at some point a quotation mark can be present in the value of GOGC.